bin/varnishd/cache/cache_cli.c

```diff
@@ -78,6 +78,7 @@ cli_cb_before(const struct cli *cli)
 	VSL(SLT_CLI, NO_VXID, "Rd %s", VSB_data(cli->cmd));
 	Lck_Lock(&cli_mtx);
 	VCL_Poll();
+	VCP_RelPoll();
```

Comment:
Out of curiosity, why not just `VCP_Poll()`?

Reply:
Maybe I'm wrong but I figured that at some point it may be possible that we poll VCPs for something else than releasing them ^^